faucet/src/main.rs

```diff
@@ -68,12 +62,13 @@ async fn main() -> Result<(), Error> {
             .try_connect(signer.clone(), shutdown_tx.clone())
             .await?;
 
+        let user_allowance: Allowance = [(100, Token(DOT)), (100, Token(IBTC)), (100, Token(INTR)), (100, ForeignAsset(1))];
```

Comment:
Does that work for KSM and kBTC as well? Note that USDT might have a different foreign asset id across chains as well.